### PR TITLE
Update documentation of class Vector.

### PR DESCRIPTION
**Pull request by bangerth**
Every once in a while one comes by a piece of documentation that has
apparently not been updated since 1998...

### CODE REVIEW

File: include/deal.II/lac/vector.h [hunk 1]
```diff
@@ -78,15 +78,22 @@ namespace parallel
...
+ * A class that represents a vector of numerical elements. As for the
+ * other classes, in the @ref Vectors group, this class has a substantial
+ * number of member functions. These include:
+ * - functions that initialize the vector or change its size;
+ * - functions that compute properties of the vector, such as a variety of
+ *   norms;
+ * - functions that allow reading from or writing to individual elements of the
+ *   vector;
+ * - functions that implement algebraic operations for vectors, such as
+ *   addition of vectors;
```
Comment (jppelteret) [inline include/deal.II/lac/vector.h]: vectors; -> vectors; and

(since this is a punctuated list)

File: include/deal.II/lac/vector.h [hunk 2]
```diff
@@ -78,15 +78,22 @@ namespace parallel
...
+ * - functions that allow inputting and outputting the data stored by vectors.
+ *
+ * In contrast to the C++ standard library class std::vector, this class
```
Comment (jppelteret) [inline include/deal.II/lac/vector.h]: <tt>std::vector</tt>

**Comment by bangerth** [comment]
Fixed.

/rebuild

**Comment by bangerth** [comment]
Not sure whether a testsuite run is warranted. If you think it's ok as is, feel free to merge.

**Comment by jppelteret** [comment]
Since its just documentation, we need not bother the tester

**Pull request merged**